src/components/Person/Person.jsx

```diff
@@ -1 +1,45 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let partner = null;
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partner = (
+        <div className="Person__partner">
+          {partnerName}
+          {' '}
+          is my wife
+        </div>
+      );
+    } else if (sex === 'f') {
+      partner = (
+        <div className="Person__partner">
+          {partnerName}
+          {' '}
+          is my husband
+        </div>
+      );
+    }
+  } else {
+    partner = <div className="Person__partner">I am not married</div>;
+  }
```

Comment:
let's just save wife/husband in some var and use it in return instead of this logic